docs/DeveloperGuide.md

```diff
@@ -550,234 +554,125 @@ Use case ends.
   1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
   Use case resumes from step 1.
 
-#### Use case: UC05 - Add an internship
+#### Use case: UC05 - Find companies
 
 Precondition: User already has an existing list of companies <br/>
-Guarantees: Addition of internship is successful
+Guarantees: Companies whose names matches the keywords specified is listed 
 
 **MSS**
 
-1.  User requests to add an internship to a company and provides the index and relevant details.
-2.  InternHunter adds the internship to the list of internships of the company. <br/>
-    Use case ends.
+1. User searches for a company name.
+2. InternHunter displays the companies sought.<br/>
+Use case ends.
 
 **Extensions**
 
  1a. InternHunter detects an error in the input format. <br/>
   1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
   Use case resumes from step 1.
 
- 1b. InternHunter detects an invalid index. <br/>
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
-  Use case resumes from step 1.
 
-#### Use case: UC06 - Delete an internship
+#### Use case: UC06 - List all companies
 
-Precondition: User already has an existing list of internships in a company <br/>
-Guarantees: Deletion of internship is successful
+Precondition: User already has an existing list of companies <br/>
+Guarantees: All companies stored in Internhunter is shown
 
 **MSS**
 
-1.  User requests to delete an internship of a company and provides the index.
-2.  InternHunter removes the internship from the list of internships in a company. <br />
-    Use case ends.
+1. User queries for all companies. 
+2. InternHunter displays all company it stores.<br/>
+Use case ends.
 
 **Extensions**
 
  1a. InternHunter detects an error in the input format. <br/>
   1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
   Use case resumes from step 1.
 
- 1b. InternHunter detects an invalid index. <br/>
-  1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
-  Use case resumes from step 1.
-  
-#### Use case: UC07 - Edit an internship
 
-Precondition: User already has an existing list of internships in a company <br/>
-Guarantees: Editing of internship is successful
+#### Use case: UC07 - Add an internship
 
-**MSS**
+Precondition: User already has an existing list of companies <br/>
```

Comment:
Should we add a guarantee for this use case as well, to standardize with all our other use cases? Same question goes for UC10, 23 and 26.

Reply:
There was a discussion where we decided against guarantee for UC26 because the user has the option to exit. I think the precondition for tabs would be has tabs but that is unnecessary to include?